### ISSUE TITLE
Edited picture is not updated while opening until browser refresh.

### ISSUE THREAD

**Issue by SergioBertolinSG** [owncloud/gallery#188]
### Steps to reproduce

1. Upload a picture.
2. Edit the picture locally and upload it.
3. Open the picture without refreshing page.

### Expected behaviour
Picture opened is the current one, not the old one.

### Actual behaviour
The edition of the file is not shown. You have to refresh the page to see the new one. (Seems like the file wasn't uploaded, but it was)

### Server configuration
**Operating system**:
Ubuntu 14.04

**Web server:**
Apache

**Database:**
MySQL

**PHP version:**
5.5.9

**ownCloud version:** (see ownCloud admin page)
{"installed":true,"maintenance":false,"version":"8.1.0.6","versionstring":"8.1 beta 2","edition":"Enterprise"}

**Updated from an older ownCloud or fresh install:**
Fresh

**List of activated apps:**
```
Enabled:
 - activity: 2.0.0
 - admin_audit: 0.6
 - encryption: 1.0.0
 - enterprise_key: 0.1.0
 - files: 1.1.9
 - files_external: 0.2.3
 - files_locking: 
 - files_pdfviewer: 0.7
 - files_sharing: 0.6.1
 - files_sharing_log: 0.5
 - files_texteditor: 0.4
 - files_trashbin: 0.6.2
 - files_versions: 1.0.5
 - files_videoviewer: 0.1.3
 - firewall: 0.91.2
 - firstrunwizard: 1.1
 - gallery: 0.6.0
 - provisioning_api: 0.2
 - templateeditor: 0.1
 - updater: 0.4
 - windows_network_drive: 0.1.1
Disabled:
 - external
 - files_antivirus
 - files_drop
 - files_ldap_home
 - objectstore
 - sharepoint
 - user_external
 - user_ldap
 - user_shibboleth
 - user_webdavauth
```

**The content of config/config.php:**

```

```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No

**Are you using encryption:** 
Yes

**Logs**
```
```

### Client configuration

**browser**
Chrome 41, Firefox

**Comment by oparoz** [comment]
This app is unmaintained, please add your request to https://github.com/owncloud/galleryplus/issues if it doesn't work with the new Gallery.
The version for 8.1 is in the app store:
https://apps.owncloud.com/content/show.php?content=169939

**Comment by oparoz** [comment]
And I can confirm that the bug is valid. The browser caches the image and the app never invalidates it.

**Comment by oparoz** [comment]
The slideshow in Gallery(+) has been fixed.

**Comment by davitol** [comment]
Testing another thing I found this issue, but the current behaviour is that the image is not shown right even after refreshing. The only way to view it well is downloading it to local and opening it.

**Comment by oparoz** [comment]
Closing as it's fixed for 8.2
